Firebase-auth depends on older hono version.

Thanks to Cloudflare and Hono It's a fun library 💯 

By the way conflicts occur when doing `npm i` locally.
Do I need to upgrade the version of hono that @hono/firebase-auth depends on?

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @hono/firebase-auth@1.1.0
npm ERR! Found: hono@3.0.2
npm ERR! node_modules/hono
npm ERR!   hono@"^3.0.2" from the root project
npm ERR!   peer hono@"^3.0.2" from @hono/sentry@0.2.0
npm ERR!   node_modules/@hono/sentry
npm ERR!     @hono/sentry@"^0.2.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: hono@2.7.8
npm ERR! node_modules/hono
npm ERR!   peer hono@"^2.7.2" from @hono/firebase-auth@1.1.0
npm ERR!   node_modules/@hono/firebase-auth
npm ERR!     @hono/firebase-auth@"^1.1.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

```
  "dependencies": {
    "@hono/sentry": "^0.2.0",
    "@hono/firebase-auth": "^1.1.0",
    "hono": "^3.0.2"
  }
```
My environment:
Node : 19.7.0
Mac OS 13.0.1

Thank you for your help in advance.

--- a/.changeset/thirty-snails-develop.md
+++ b/.changeset/thirty-snails-develop.md
@@ -1,0 +1,5 @@
+---
+'@hono/firebase-auth': minor
+---
+
+feat: support Hono v3

--- a/packages/firebase-auth/package.json
+++ b/packages/firebase-auth/package.json
@@ -31,13 +31,13 @@
     "firebase-auth-cloudflare-workers": "^1.0.0"
   },
   "peerDependencies": {
-    "hono": "^2.7.2"
+    "hono": "3.*"
   },
   "devDependencies": {
-    "hono": "^2.7.2",
     "@cloudflare/workers-types": "^3.14.1",
     "@types/jest": "^28.1.4",
     "firebase-tools": "^11.4.0",
+    "hono": "^3.0.3",
     "jest": "^28.1.2",
     "jest-environment-miniflare": "^2.6.0",
     "prettier": "^2.7.1",

--- a/packages/firebase-auth/src/index.ts
+++ b/packages/firebase-auth/src/index.ts
@@ -1,10 +1,11 @@
-import type { EmulatorEnv, KeyStorer, FirebaseIdToken } from 'firebase-auth-cloudflare-workers'
+import type { KeyStorer, FirebaseIdToken } from 'firebase-auth-cloudflare-workers'
 import { Auth, WorkersKVStoreSingle } from 'firebase-auth-cloudflare-workers'
 import type { Context, MiddlewareHandler } from 'hono'
 
-export interface VerifyFirebaseAuthEnv extends EmulatorEnv {
+export type VerifyFirebaseAuthEnv = {
   PUBLIC_JWK_CACHE_KEY?: string | undefined
   PUBLIC_JWK_CACHE_KV?: KVNamespace | undefined
+  FIREBASE_AUTH_EMULATOR_HOST: string | undefined
 }
 
 export interface VerifyFirebaseAuthConfig {
@@ -22,12 +23,6 @@ const defaultKeyStoreInitializer = (c: Context): KeyStorer => {
     c.env.PUBLIC_JWK_CACHE_KEY ?? defaultKVStoreJWKCacheKey,
     c.env.PUBLIC_JWK_CACHE_KV
   )
-}
-
-type Env = {
-  Bindings: {
-    FIREBASE_AUTH_EMULATOR_HOST: string
-  }
 }
 
 export const verifyFirebaseAuth = (userConfig: VerifyFirebaseAuthConfig): MiddlewareHandler => {

